cvmfs/receiver/commit_processor.cc

```diff
@@ -73,7 +78,8 @@ CommitProcessor::Result CommitProcessor::Process(
 
   // Current catalog from the gateway machine
   if (!manifest.IsValid()) {
-    LogCvmfs(kLogCvmfs, kLogStderr, "Could not open repository manifest");
+    LogCvmfs(kLogReceiver, kLogCustom1,
+             "Could not open repository manifest\n");
```

Comment:
Ah, sorry, did I forget to add the newline inside `LogCvmfs`?  Sorry about that!  I'll fix it.